src/domain/_common/components/VariantButton.tsx

```diff
@@ -2,8 +2,8 @@ import React, { type ReactNode } from 'react';
 
 interface ButtonProps {
   isDisabled: boolean;
-  type: 'normal' | 'text' | 'primary' | 'large';
-  onClick: () => void;
+  style: 'normal' | 'text' | 'primary' | 'large';
+  type?: 'button' | 'submit';
```

Comment:
```suggestion
  type?: 'button' | 'submit';
  onClick?: () => void;
```
onClick 핸들러가 살아있긴 해야할 것 같아요.
`type == 'button'` 일 때는 버튼에 클릭 이벤트 정의해야하니.. 대신 optional로요~

Reply:
onClick 핸들러는 다시 살릴게요. 제가 생각해봐도 없으면 안될 것 같네요